### ISSUE TITLE
// is being removed from comments

### ISSUE THREAD

**Issue by vjeux** [prettier/prettier#1537]
```js
/* http://www.site.com or http://www.site.com/page.html */
```

outputs

```js
/* http:www.site.com or http://www.site.com/page.html */
```

**Comment by azz** [comment]
Should be fixed over at eslint/typescript-eslint-parser/issues/256

**Comment by JamesHenry** [comment]
A fix for this was merged in the parser today: https://github.com/eslint/typescript-eslint-parser/pull/219

**Comment by azz** [comment]
Nice.

@despairblue Do you have this fix in your local work?

**Comment by despairblue** [comment]
@azz nope, not yet. I hope to squeeze some free time out off my schedule tonight.

**Comment by despairblue** [comment]
@azz it's fixed in my WIP PR #1578
